src/app/doc_exporter.cpp

```diff
@@ -1236,8 +1242,10 @@ Doc* DocExporter::createEmptyTexture(const Samples& samples,
       maxColors,
       m_docBuf));
 
-  if (palette)
-    sprite->setPalette(palette, false);
+  if (palette.size() == 0)
+    // Old behavior
+    palette.resize(256);
+  sprite->setPalette(&palette, false);
```

Comment:
Is this needed? or can we do this:
```suggestion
  if (palette.size() > 0)
    sprite->setPalette(&palette, false);
```